start-a-local-cluster.md

```diff
@@ -6,21 +6,21 @@ toc: false
 Once you've [installed CockroachDB locally](/install-cockroachdb), the quickest way to test out the database is to start a single-node cluster and talk to the node via the built-in SQL client. There are two ways to do this:
 
 - [Development Mode (Insecure)](#development-mode)  
-Data is stored in-memory and client/server communication is completely insecure. This mode is great for learning CockroachDB functionality but not suitable for actual development.
+Data is stored in-memory and client/server communication is completely insecure. This mode is great for learning CockroachDB, but there's no authentication or encryption, so it's not suitable for actual development.
 
 - [Standard Mode (Secure)](#standard-mode)  
-Data is stored on-disk and client/server communication is secure. Setup involves creating certificates and passing certain command line options, but it's still simple. This mode is safest for actual development. 
+Data is stored on-disk and client/server communication is secure. Setup involves creating certificates and passing certain command line options, but it's still simple. This mode is suitable for actual development. 
```

Comment:
"suitable for development" is kind of specific and not really what people will have in mind. Why not say something more like, "This mode is suitable for standing up a persistent test cluster to develop an application or test CockroachDB."